desc/compute/_profiles.py

```diff
@@ -1065,13 +1236,13 @@ def _iota_den_r(params, transforms, profiles, data, **kwargs):
             data["omega_t"]
             * data["g_tz_r"]
             * data["sqrt(g)_rr"]
-            / (2 * data["sqrt(g)_r"] ** 2)
+            / (2 * data["sqrt(g)_r"] ** 2)  # XXX: this factor of 2 not in Eq. 48
```

Comment:
This computes what the docstring says. The extra factor of 2 is in $d \alpha / d \rho$, $d \beta / d \rho$, and $d \gamma / d \rho$, and hence cancels out in [equation 48](https://github.com/PlasmaControl/DESC/files/12137234/rotational_transform_and_limit.pdf). Since the code computes equation 49, it is simplest to keep the factors of 2 and compute the limit as in equation 50.